cheat-sheet.md

```diff
@@ -233,28 +233,51 @@ Download agent logs
 Media Controller API for chat - ***New,ChatJS Required***
 
     connect.contact(function(contact){
-    contact.getAgentConnections().forEach(function(connection){
-      // chat users
-      if(connection.getMediaType() === connect.MediaType.CHAT){
-          contact.getConnection().getMediaController()
-          .then(function(controller){
-              controller.onMessage(function(response){
-                  console.log("data", response)
-              })
-
-              controller.sendMessage({message: "so and so", contentType: "text/plain"})
-                  .then(function(res, req){
-                    console.log(res.status)
-              });
-              
-              controller.onDisconnect(function(){
-                  console.log("on disconnect");
-              })
-          });
-      }
+	// WE HAVE .getAgentConnection() [Singular] and not .getAgentConnections() [Plural]
+	// We fetch agentId from .getAgentConnection() to compare with the connections that we would get from .getConnections()
+	
+	var connectionId = contact.getAgentConnection()['connectionId']
+	
+	// WE HAVE .getConnections() [Plural] and not .getConnection() [Singular]
+        //It returns an array with connection between the AGENT and CUSTOMER 
+	
+	contact.getConnections().forEach(function (connection) {
+	//console.log(connection)
+	
+	//connection.getMediaType() === chat , AND  !== connect.MediaType.CHAT
+        // connection.getMediaType() returns "chat" and not connect.MediaType.CHAT
```

Comment:
connect.MediaType.CHAT is defined as "chat"

Reply:
See [here](https://github.com/amazon-connect/amazon-connect-streams/blob/f046b812073daadecc9370ae4649e95990d818e3/src/api.js#L136)